internal/services/springcloud/migration/accelerator_customized_migration_v0_to_v1.go

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package migration
+
+import (
+	"context"
+	"log"
+
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/springcloud/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+)
+
+type SpringCloudCustomizedAcceleratorV0ToV1 struct{}
+
+func (s SpringCloudCustomizedAcceleratorV0ToV1) Schema() map[string]*pluginsdk.Schema {
+	return map[string]*pluginsdk.Schema{
+		"name": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+		},
+
+		"spring_cloud_accelerator_id": {
+			Type:     pluginsdk.TypeString,
+			Required: true,
+			ForceNew: true,
+		},
+
+		"git_repository": {
+			Type:     pluginsdk.TypeList,
+			Required: true,
+			MaxItems: 1,
+			Elem: &pluginsdk.Resource{
+				Schema: map[string]*pluginsdk.Schema{
+					"url": {
+						Type:     pluginsdk.TypeString,
+						Required: true,
+					},
+
+					"basic_auth": {
+						Type:          pluginsdk.TypeList,
+						Optional:      true,
+						ForceNew:      true,
+						MaxItems:      1,
+						ConflictsWith: []string{"git_repository.0.ssh_auth"},
```

Comment:
The state migration schemas don't require attributes like `ForceNew`, `MaxItems`, `MinItems`, `ConflictsWith`, `Default`, `ExactlyOneOf`, `RequiredWith` etc. since these aren't needed for deserializing the data. Can you please remove these from the schema?